qa/rpc-tests/proxy_test.py

```diff
@@ -83,13 +83,13 @@ def setup_nodes(self):
         # Note: proxies are not used to connect to local nodes
         # this is because the proxy to use is based on CService.GetNetwork(), which return NET_UNROUTABLE for localhost
         args = [
-            ['-listen', '-debug=net', '-debug=proxy', '-proxy=%s:%i' % (self.conf1.addr),'-proxyrandomize=1'], 
```

Comment:
yes - proxy_test does a lot of work in the `__init__()` method, which is called before the temp directory is set up. Logging requires the temp directory, so we can't use `self.log` until we're in `main()` and `_start_logging()` has been called.

I left this as a `print()` rather than trying to do anything clever to coerce the test into being log friendly.